apps/nrm/behaviours/modify-summary-page.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-len  */
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  locals(req, res, next) {
+    const locals = super.locals(req, res, next);
+    locals.rows.map(row => {
+      row.fields.map(field => {
+        // add hint under reference label on summary pages
+        if (field.field === 'reference') {
+          field.referenceLabelHint = true;
+        }
+        // add text under pv-want-to-submit-nrm label and omit change link on summary pages if automatic referral is true
+        if (field.field === 'pv-want-to-submit-nrm') {
+          if (field.value === 'Yes' && req.sessionModel.get('automatic-referral') === true) {
+            field.omitChangeLink = true;
+            field.automaticReferral = true;
+          }
+        }
+        // add text under user-email label on summary pages for first responder details
+        if (field.step === '/fr-details' && field.field === 'user-email') {
+          field.emailInfo = true;
+        }
+      });
+      // change organisation label in Your details on summary pages
+      if (row.section === 'Your details') {
+        row.fields.map(field => {
+          if (field.label === 'Your organisation') {
+            field.label = 'Organisation';
+          }
+        });
+        return row;
+      }
+      return row;
+    });
+    return locals;
+  }
+};
```

Comment:
I like the comments

In your code you're using map for Side Effects.  A map is usually used for transforming and returning a new array.  `forEach` might be more appropriate. Also `forEach` is better for performance for larger datasets. 

This is just a suggestion, so you can ignore

```
'use strict';

module.exports = superclass => class extends superclass {
  locals(req, res, next) {
    const locals = super.locals(req, res, next);

    locals.rows.forEach(row => {
      row.fields.forEach(field => {
        this.enhanceFieldForSummary(row, field, req.sessionModel);
      });
    });

    return locals;
  }

  enhanceFieldForSummary(row, field, sessionModel) {
    if (field.field === 'reference') {
      field.referenceLabelHint = true;
    } else if (field.field === 'pv-want-to-submit-nrm') {
      if (field.value === 'Yes' && sessionModel.get('automatic-referral') === true) {
        field.omitChangeLink = true;
        field.automaticReferral = true;
      }
    } else if (field.step === '/fr-details' && field.field === 'user-email') {
      field.emailInfo = true;
    }

    // Modify 'Your organisation' label conditionally (optional)
    if (row.section === 'Your details' && field.label === 'Your organisation') {
      field.label = 'Organisation';
    }
  }
};
```
Note: you might need to update the code above as I changed it a while ago and I see your code has been updated

Reply:
I see what you mean...I have updated the behaviour to use forEach as suggested